src/Layout/Flex/index.tsx

```diff
@@ -12,14 +12,24 @@ type Props = FlexProps &
 
 export const StyledFlex = styled(Box)<Props>`
   display: flex;
-  flex-direction: ${({ flexDirection }) =>
-    flexDirection || FlexPropAndDefaultValueMap.flexDirection};
-  justify-content: ${({ justifyContent }) =>
-    justifyContent || FlexPropAndDefaultValueMap.justifyContent};
-  align-items: ${({ alignItems }) =>
-    alignItems || FlexPropAndDefaultValueMap.alignItems};
-  flex-wrap: ${({ flexWrap }) =>
-    flexWrap || FlexPropAndDefaultValueMap.flexWrap};
+  flex-direction: ${({ flexDirection }) => flexDirection};
+  justify-content: ${({ justifyContent }) => justifyContent};
+  align-items: ${({ alignItems }) => alignItems};
+  flex-wrap: ${({ flexWrap }) => flexWrap};
 `;
 
-export const Flex: React.FC<Props> = props => <StyledFlex {...props} />;
+export const Flex: React.FC<Props> = ({
+  flexDirection = 'row',
+  justifyContent = 'flex-start',
+  alignItems = 'stretch',
+  flexWrap = 'nowrap',
+  ...restProps
```

Comment:
Should there be a description of what restProps is at the top and to what component they are passed?

Reply:
The rest of the props have been listed in the control table of Flex story as well. But I also add the comment in the codebase. 🙏